src/ssda/ssda_populate.py

```diff
@@ -144,17 +177,14 @@ def validate_options(
         raise click.UsageError("The start date must be earlier than the end date.")
```

Comment:
The error message is not consistent with the condition checked. Also, there should be an additional check that the start date is earlier than the end date.